lib/appoptics_apm.rb

```diff
@@ -59,7 +58,7 @@
   else
     $stderr.puts '=============================================================='
     $stderr.puts 'AppOpticsAPM not loaded. Tracing disabled.'
-    $stderr.puts 'Service Key may be wrong or missing.'
+    $stderr.puts 'Service Key or other settings may be wrong or missing.'
```

Comment:
i just recently went the other direction of being more specific about errors. as i user it's often frustrating to see "or other settings" because it doesn't help me at all. (maybe a more explicit message has already been issued below though.)

Reply:
good point. In this case there should be a message logged before to indicate the problem. Let me think about a phrase that encourages users to look at previous log messages.